{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf320
{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fswiss\fcharset0 ArialMT;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww9000\viewh8400\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\fs24 \cf0 Bugreports:\
\
Foodputer:\
\pard\pardeftab720\ql\qnatural

\f1\fs26 \cf0 Traceback (most recent call last):\
\'a0File "/opt/drinkomatic/micropay.py", line 11, in <module>\
\'a0 \'a0barcode_reader = serial.Serial("/dev/ttyUSB0", baudrate=9600)\
\'a0File "/usr/lib/python2.6/dist-packages/serial/serialutil.py", line 166, in __init__\
\'a0 \'a0self.open()\
\'a0File "/usr/lib/python2.6/dist-packages/serial/serialposix.py", line 175, in open\
\'a0 \'a0raise SerialException("could not open port %s: %s" % (self._port, msg))\
serial.serialutil.SerialException: could not open port /dev/ttyUSB0: [Errno 5] Input/output error: '/dev/ttyUSB0'\
\
\
Labelprinter:\
okay jeg ved ikke om jeg f\'e5r det hele med men.\
systemet gik ned vilke var grunde til at vi stoppede med at s\'e6tte koder p\'e5 ig\'e5r.\
der mangler en m\'e5de g\'e5 tilbage og se om tingende er blevet f\'f8rt\
rigtigt ind i databasen,\
der mangler ogs\'e5 en m\'e5de at f\'e5 den til at printe igen, for det var\
ikke altid der kom labels ud n\'e5r den skulle printe,\
databasen er ikke koplet op til noget, s\'e5 selv om folk har programmer\
til at l\'e6se QR koder p\'e5 deres telefon kan de bruge det til noget,\
der skal ogs\'e5 kunne tages billede og ligges sammen p\'e5 hvad der godt\
kunne v\'e6re en wiki side hvor informationen om den ting QR coden sidder\
p\'e5.\
s\'e5 man hved hvordan tingende ser ud\
\
og s\'e5 er der lige nu 3 piktogrammer man kan s\'e6tte under sin tekst, der\
skal muligvis flere til, om ikke andet en defination af had de tre\
betyder, (DNH siger sig selv men de andre to )\
}